addons/l10n_ec/data/account_tax_tag_data.xml

```diff
@@ -0,0 +1,975 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data>
+        <record id="tax_report_104" model="account.tax.report">
+            <field name="name">Reporte 104</field>
+            <field name="country_id" ref="base.ec" />
+        </record>
+        <record id="tax_tag_303" model="account.tax.report.line">
+            <field name="name">303</field>
+            <field name="code">303</field>
```

Comment:
It seems like the tax tags were translated quickly into account.tax.report.line.  We should not call the file account_tax_tag_data.xml anymore but tax_report something.  Also, we should provide proper names for the lines, such that if someone opens the report, he can understand something.  (there is probably some hierarchy to be added as well)